internal/callgraph/language/java11/cmd_factory_test.go

```diff
@@ -36,3 +39,42 @@ func TestMakeCallGraphGenerationCmd(t *testing.T) {
 	assert.Contains(t, args, "targetclasses")
 	assert.Contains(t, args, "dependencypath")
 }
+
+func TestMakeBuildMavenCmd(t *testing.T) {
+	jarPath := "jarpath"
+	ctx, _ := ctxTestdata.NewContextMock()
+	cmd, err := CmdFactory{}.MakeBuildMavenCmd(jarPath, ctx)
+
+	assert.NoError(t, err)
+	assert.NotNil(t, cmd)
+	args := cmd.Args
+	assert.Contains(t, args, "mvn")
+	assert.Contains(t, args, "package")
+	assert.Contains(t, args, "-q")
+	assert.Contains(t, args, "-DskipTests")
+}
+
+func TestMakeBuildMavenCmdFunctional(t *testing.T) {
```

Comment:
This test causes a slowdown which may not be acceptable for the main test suite, but I believe it's important to actually test that the build command runs as it should in the correct directory. Let me know if I should move this test.

Reply:
It is a significant slowdown sadly. 
I think we definitely should move it outside of the unit-tests if possible, maybe into the test folder in the project root? I would suggest discussing with @viktigpetterr around testing external commands, since we do not do that at the moment, except for the single resolve test in the root test folder which tests the whole execution of a successful resolve.

Reply:
I agree, slowing the unit tests down this much will interfere with development speed. After talking to @viktigpetterr, we decided that I should move this test to a separate testing location for end-to-end tests and such. I believe it should be moved here: [/test/resolve](https://github.com/debricked/cli/tree/main/test/resolve)